src/bci_build/package/mariadb/README.server.md.j2

```diff
@@ -0,0 +1,66 @@
+# {{ image.pretty_name }} Container Image
+
+MariaDB is one of the most popular database servers in the world. It's made by the original developers of MySQL and guaranteed to stay open source.
```

Comment:
imho leave this out. SUSE should stay neutral, we also offer postgresql and in future might also be offering further things

Reply:
I replaced this with the RPM package description.